ptop.py

```diff
@@ -0,0 +1,115 @@
+import time
+import os
+import argparse
+
+parser = argparse.ArgumentParser(
+    prog="tailf",
+    description="Reads and prints the last 10 lines of a file line by line every 10 seconds",
+)
+
+parser.add_argument("logfile", help="Path to the log file")
+parser.add_argument(
+    "--total-jobs", "-tj", dest="tj", type=int, help="The number of jobs in the order"
+)
+args = parser.parse_args()
+tj = args.tj
+
+
+def clear_console():
+    os.system("cls" if os.name == "nt" else "clear")
```

Comment:
```suggestion
    print("\033[H\033[J", end="")
```

Can you try, if this works in the windows shell as well? If it works, let's use this as it is more efficient than executing an external command.